iot-e2e-tests/android/buildAndRunTests.cmd

```diff
@@ -0,0 +1,7 @@
+call gradlew :app:assembleDebug :app:assembleDebugAndroidTest
+python AndroidDeviceSelect.py
+set /p device=<devcie.txt
+ECHO We're working with "%device%"
+call adb -s %device% install -r -g  app\build\outputs\apk\app-debug.apk
+call adb -s %device% install -r -g  app\build\outputs\apk\app-debug-androidTest.apk
```

Comment:
What if this step fails ? how is the user informed of errors ?

Reply:
We usually maintain logs so that when we go back and look at it, we know whats failing to fix. So please add failures everywhere